benchmark/server/scenarios/system.js

```diff
@@ -12,74 +11,42 @@ const suite = new Benchmark.Suite('system', {
 });
 Benchmark.options.minSamples = 100;
 
-const materialSystem = compose(palette, spacing, typography);
-const styledSystem = compose2(color, space, fontFamily, fontSize);
-
 const materialSystemTheme = createMuiTheme();
 
-const styledSystemTheme = createMuiTheme();
-styledSystemTheme.breakpoints = ['40em', '52em', '64em'];
-styledSystemTheme.colors = styledSystemTheme.palette;
-styledSystemTheme.fontSizes = styledSystemTheme.typography;
-styledSystemTheme.fonts = styledSystemTheme.typography;
+const styledSystemTheme = {
+  breakpoints: ['40em', '52em', '64em'],
+  colors: {
+    primary: materialSystemTheme.palette.primary,
+    background: materialSystemTheme.palette.background,
+  },
+  fontSizes: materialSystemTheme.typography,
+  fonts: materialSystemTheme.typography,
+};
+
+const styledSystemCSSFunction = css({
```

Comment:
Was a bit tricky to me what needs to be sent as a param where, but it should be ok now - styles go to css, the theme to the returned function.